drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -119,8 +119,33 @@ void TM_EvtReceivedCb(TL_EvtPacket_t *hcievt)
 					     K_FOREVER);
 			break;
 		}
+/*
+ * https://github.com/zephyrproject-rtos/zephyr/issues/19509
+ * For some reason BLE stack (v 1.2.0) is not reporting the peer address
+ * 0A 00 01 08 01 01 FF FF FF FF FF FF 00 00 00 00 00 00 0C AA C5 B3 3D 6B ...
+ * If such message is passed to HCI core than no Secure connection could be
+ * established. Temporary copy RPA to PEER address
+ */
+		struct bt_hci_evt_le_meta_event *mev;
```

Comment:
With all the 80 character limit violations you might consider just moving this into a function.